frontend/packages/knative-plugin/console-extensions.json

```diff
@@ -1124,5 +1124,407 @@
     "flags": {
       "required": ["KNATIVE_SERVING_SERVICE", "KNATIVE_SERVING_REVISION", "KNATIVE_EVENTING"]
     }
+  },
+  {
+    "type": "console.model-metadata",
+    "properties": {
+      "model": {
+        "group": "sources.knative.dev"
+      },
+      "color": "#7d1007"
+    }
+  },
+  {
+    "type": "console.model-metadata",
+    "properties": {
+      "model": {
+        "group": "serving.knative.dev"
+      },
+      "color": "#005f60"
+    }
+  },
+  {
+    "type": "console.model-metadata",
+    "properties": {
+      "model": {
+        "group": "operator.knative.dev"
+      },
+      "color": "#005f60"
+    }
+  },
```

Comment:
at other occurrences it looks good to have color with the group for serving and eventing however here we can avoid as both `knativeServing` and `knativeEventing` has a same group so better to specify kind as well explicitly